ui/page/components/wallet_account_selector.go

```diff
@@ -291,16 +291,19 @@ func (ws *WalletAndAccountSelector) Layout(window app.WindowNavigator, gtx C) D
 					}.Layout(gtx,
 						layout.Rigid(ws.setWalletLogo),
 						layout.Rigid(func(gtx C) D {
+							gtx.Constraints.Max.X = gtx.Constraints.Max.X * 3 / 5
```

Comment:
```suggestion
							gtx.Constraints.Max.X = gtx.Constraints.Max.X * 2.5 / 5
```

let's reduce the max length a little